<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modifypage</class>
 <widget class="QWidget" name="Modifypage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="c_t_name">
        <property name="styleSheet">
         <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>电话号码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>电子邮件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>学校名称：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="c_t_phonenum">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="c_t_email">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="c_t_school">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_classmate">
      <property name="text">
       <string>修改到同学</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="r_t_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>电话号码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>电子邮件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>称呼：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QTextEdit" name="r_t_phonenum">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="r_t_email">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="r_t_address">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_relative">
      <property name="text">
       <string>修改到亲戚</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="f_name">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="f_t_name">
        <property name="styleSheet">
         <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="f_phonenum">
        <property name="text">
         <string>电话号码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="f_email">
        <property name="text">
         <string>电子邮件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="f_kplace">
        <property name="text">
         <string>相识地点：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="f__t_phonenum">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="f_t_email">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="f_t_kplace">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_friend">
      <property name="text">
       <string>修改到朋友</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="w_t_name">
        <property name="styleSheet">
         <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>电话号码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>电子邮件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>公司名称：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextEdit" name="w_t_phonenum">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="w_t_email">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="w_t_company">
        <property name="styleSheet">
         <string notr="true">font: 25 9pt &quot;微软雅黑 Light&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_workmate">
      <property name="text">
       <string>修改到同事</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
